vehicle/tests/golden/error/tensorLike/nonMatchingRecordFields/TypeCheck.err.golden

```diff
@@ -0,0 +1,2 @@
+Error in file 'spec.vcl' at Line 4, Columns 8-13: . In particular 'Real' is not equal to 'Nat'.
+Fix: check your types
```

Comment:
Yup before we merge this, we should create an issue to improve this error message. It should mention that this constraint is generated because of the @tensor annotation.